.github/ISSUE_TEMPLATE/02-feature.yaml

```diff
@@ -1,6 +1,6 @@
 name: 🚀 Feature request
 description: Suggest an idea for this project 💡!
-labels: ["type: feature request"]
+type: Enhancement
```

Comment:
We currently have in the label space `type: enhancement` and `type: feature request`, each with different meaning, namely "improve existing functionality" and "suggest or ask new feature", respectively. Assuming we want to have those 2 concepts captured in issue types (I would vote for it) It would be good to preserve this separation by naming the `type` here with `Feature` or something similar, but not `Enhancement`.

Reply:
We have to set the allowed types at the organisation level (if I understand the docs correctly), but that seems easily doable.